### PR TITLE
Fix the sensitivity calculations for repeated calls

### PR DESCRIPTION
**Pull request by jamiebramwell**
This fixes a bug where the old material parameter finite element state was referenced when a sensitivity calculation was requested on a new finite element space.

### CODE REVIEW

**Comment by jamiebramwell** [comment]
ping @junkudo

File: src/serac/physics/tests/serac_solid_sensitivity.cpp [hunk 1]
```diff
@@ -7,7 +7,7 @@
 #include <mfem.hpp>
 #include <memory>
 
-TEST(SeracFEState, finiteDiff)
+TEST(serac_solid_sensitivity, finite_diff)
```
Comment (white238) [inline src/serac/physics/tests/serac_solid_sensitivity.cpp:10]: 👍

**Review by white238 — APPROVED** [review]

**Comment by jamiebramwell** [comment]
@jandrej @samuelpmishLLNL could you take a look? This one is pretty small. Thanks!

**Comment by jamiebramwell** [comment]
@samuelpmishLLNL , can you take a look at this? Thanks!

**Review by samuelpmishLLNL — APPROVED** [review]
Sorry, I completely forgot about this PR.

**Comment by jamiebramwell** [comment]
> Sorry, I completely forgot about this PR.

So did I! 😆

**Pull request merged**